src/components/Alert.js

```diff
@@ -7,7 +7,7 @@ export default function Alert({ alertIcon, alertMessage }) {
     let timeoutId;
     if (isVisible) {
       timeoutId = setTimeout(() => {
-        setAlert(false);
+        setIsVisible(false);
```

Comment:
Something is wrong here, the delete confirmation does not disappear after 5 seconds.

Reply:
👍

Reply:
It will be fixed in the ticket
Bug fix: Deletion Alert is not disappearing after a set amount of time